compiler/hie-core/test/src/Development/IDE/Test.hs

```diff
@@ -52,9 +52,12 @@ expectDiagnostics expected = do
                   diagsNot <- skipManyTill anyMessage LspTest.message :: Session PublishDiagnosticsNotification
                   let fileUri = diagsNot ^. params . uri
                   case Map.lookup (diagsNot ^. params . uri . to toNormalizedUri) m of
-                      Nothing -> liftIO $ assertFailure $
-                          "Got diagnostics for " <> show fileUri <>
-                          " but only expected diagnostics for " <> show (Map.keys m)
+                      Nothing -> do
+                          let actual = diagsNot ^. params . diagnostics in
```

Comment:
```suggestion
                          let actual = diagsNot ^. params . diagnostics
```
This seems to be a compile error atm

Reply:
Thanks, yes forgot to remove the `in` when shuffling lines around.